cockatrice/src/window_main.cpp

```diff
@@ -1030,6 +1030,10 @@ void MainWindow::actAddCustomSet()
         QMessageBox::warning(this, tr("Load sets/cards"), tr("Selected file cannot be found."));
         return;
     }
+    else if (QFileInfo(fileName).suffix() != "xml") { // fileName = *.xml
```

Comment:
Just a minor remark: no need to `else`, since the previous `if` block contains a `return`.